src/main/java/Application.java

```diff
@@ -0,0 +1,88 @@
+package src.main.java;
+
+import java.util.Scanner;
+
+public class Application {
+	private ATM atm = new ATM(this);
+	private boolean exit = false;
+	private Scanner scan = new Scanner(System.in);
+	
+	public Application() {
+		StartMenuRunning();
+	}
+	
+	public void StartMenuRunning() {
+		while(!this.exit) {
+			Display.Menu();
+			String name = scan.nextLine();
+			ATMaccount account = atm.getAccountUser(name);
+			if(account != null) runningAccount(account);
```

Comment:
Regarding this `if` statement, I would suggest to follow the [advice](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used) of the Google code style convention.

Reply:
so with bracets ?